### ISSUE TITLE
Field's names contain underscore between a letter and a number? 

### ISSUE THREAD

**Issue by p581581** [go-xorm/xorm#837]
When field's names contain underscore between a letter and a number cause "無效的資料行名稱" error, 
but it works fine when underscore between letters

For example:

Table Mark
- mark_1 VARCHAR(1)
- mark_2 VARCHAR(1)
- mark_a VARCHAR(1)
`
type Mark struct {
        Mark1       string  `xorm:"VARCHAR(1)"`
	Mark2       string  `xorm:"VARCHAR(1)"`
	MarkA       string  `xorm:"VARCHAR(1)"`
}
`
and then use `find` in my handler:
`
var marks Mark[]
err := engine.find(&marks)
`
then it throws an error "無效的資料行名稱" Mark2.

please, help.

**Comment by lunny** [comment]
Because mark_1 -> Mark1 but Mark1 -> mark1.

**Comment by p581581** [comment]
yeah I know the rule, but is there another way to fix it and keep underscores, like field's name mapping?

**Comment by lunny** [comment]
You can write yourself mapper or you can give a specify tag as a column name.
```
type Mark struct {
        Mark1       string  `xorm:"VARCHAR(1) mark_1"`
	Mark2       string  `xorm:"VARCHAR(1) mark_2"`
	MarkA       string  `xorm:"VARCHAR(1) mark_a"`
}
```
